dask/dataframe/numeric.py

```diff
@@ -0,0 +1,70 @@
+import pandas as pd
+from pandas.core.dtypes.common import is_scalar as pd_is_scalar
+
+from ..utils import derived_from
+from ..delayed import delayed
+from ..array import Array
+from .core import Series
+
+
+__all__ = ("to_numeric",)
+
+
+@derived_from(pd)
+def to_numeric(arg, errors="raise", downcast=None, meta=None):
+    """
+    Return type depends on input. Delayed if scalar, otherwise same as input.
+    """
+    if downcast not in (None, "integer", "signed", "unsigned", "float"):
```

Comment:
Thoughts on checking these here vs. letting pandas catch issues when the graph is executed? I'm not sure if we're consistent.

The upside of checking them here is that the errors are caught earlier and the stacktrace is nicer. The downside is that if pandas adds a new option, we'll need to explicitly add it here.

Reply:
Right, I was thinking that the early failure is worth the duplication, and there already need to be changes if pandas adds options.